resources/windows/msi/main.wxs.erb

```diff
@@ -0,0 +1,781 @@
+<?xml version='1.0' encoding='windows-1252'?>
```

Comment:
Let's keep this in line with other templates and name this file `project.wxs.erb`, and then rename it when we process it to be `puppet-agent.wxs` or `pe-client-tools.wxs`, whatever project we're building.

Notice we're doing this for nuget with `erb_file(File.join(VANAGON_ROOT, "resources/windows/nuget/project.nuspec.erb"), File.join(workdir, "#{name}.nuspec"), false, { :binding => binding })`